src/main/java/com/tekenable/config/AuditConfigurator.java

```diff
@@ -10,26 +14,24 @@
 
```

Comment:
Nope we can't. Or actually we can but with no effect as data definition operations are not reversible. Only insert/update/delete statements can be rolled back.